scripts/install_gen_option_001_docker.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+ORIG_DIR=${PWD}
```

Comment:
(FYI, there's also `pushd` and `popd`)

Reply:
Yes. However, please note that pushd and popd bind you to full bash, whereas alternatives like this or `cd -` work in mere sh and might thus be preferable in certain contexts.

Reply:
Ah, I did not know that!  Sounds good.